nikola/plugins/compile/markdown/__init__.py

```diff
@@ -89,8 +90,12 @@ def set_site(self, site):
         site_extensions = self.site.config.get("MARKDOWN_EXTENSIONS")
         self.config_dependencies.append(str(sorted(site_extensions)))
         extensions.extend(site_extensions)
+
+        site_extension_configs = self.site.config.get("MARKDOWN_EXTENSION_CONFIGS", {})
+        self.config_dependencies.append(json.dumps(site_extension_configs, sort_keys=True))
```

Comment:
Does it actually work like this, without actually calling the translatable setting?

Reply:
Forget it, this approach is totally broken. Looks like we need one markdown converter per-language.